vendors/twtg/drivers/neon-vs-mt-xx-xx/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "neon-vs-mt-xx-xx",
+  "version": "1.0.0-SNAPSHOT",
+  "description": "NEON Temperature Transmitter driver",
```

Comment:
This is not a temperature transmitter driver, it is NEON Multi Turn Valve driver

Reply:
Corrected